tools/ExtensionsMetadataGenerator/src/ExtensionsMetadataGenerator/ExtensionsMetadataGenerator.csproj

```diff
@@ -32,7 +32,7 @@
 
   <Target Name="UpdateRuntimeAssemblies" BeforeTargets="Build">
     <Exec Command="pwsh ./updateruntimeassemblies.ps1" Condition=" '$(OS)' == 'Unix' "/>
-    <Exec Command="powershell.exe –command .\updateruntimeassemblies.ps1" Condition=" '$(OS)' == 'Windows_NT' "/>
+    <Exec Command="powershell.exe -command .\updateruntimeassemblies.ps1" Condition=" '$(OS)' == 'Windows_NT' "/>
```

Comment:
no actual change?

Reply:
the character! I don't know what it was before, but now its a hyphen lol. You can see the slightest length difference in the diff.

Before: `0x2013`, after: `0x2D`

Reply:
Hahaha my eyes are not that good